test/hypothesis/master/test_cluster_master.py

```diff
@@ -0,0 +1,26 @@
+from hypothesis import given
+from hypothesis.strategies import text, dictionaries, integers
+from test.framework.base_unit_test_case import BaseUnitTestCase
+
+from app.master.build import Build
+from app.master.cluster_master import ClusterMaster
+
+
+class TestClusterMaster(BaseUnitTestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.patch('app.util.fs.create_dir')
+        self.patch('app.util.fs.async_delete')
+        self.patch('os.makedirs')
+
+    @given(dictionaries(text(), text()))
+    def test_handle_request_for_new_build(self, build_params):
```

Comment:
Are these tests just that the code doesn't raise an exception? If so, how do you feel about naming the tests something to indicate that -- something like `test_handle_request_for_new_build_does_not_raise_exception`.